src/test/java/uk/gov/companieshouse/chdorderconsumer/service/CHDOrderServiceTest.java

```diff
@@ -0,0 +1,115 @@
+package uk.gov.companieshouse.chdorderconsumer.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.handler.order.chd.PrivateChdOrderResourceHandler;
+import uk.gov.companieshouse.api.handler.order.chd.request.ChdOrdersPost;
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.api.model.order.chd.MissingImageDeliveryRequestApi;
+import uk.gov.companieshouse.chdorderconsumer.exception.ServiceException;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class CHDOrderServiceTest {
+
+    @InjectMocks
+    private CHDOrderService chdOrderService;
+
+    @Mock
+    private ApiClientService mockApiClientService;
+
+    @Mock
+    private InternalApiClient mockApiClient;
+
+    @Mock
+    private PrivateChdOrderResourceHandler mockPrivateChdOrderResourceHandler;
+
+    @Mock
+    private ChdOrdersPost mockChdOrdersPost;
+
+    @Mock
+    private ApiResponse<MissingImageDeliveryRequestApi> mockApiResponse;
+
+    private static final String POST_MISSING_IMAGE_CHD_ORDER_URI = "/chd-order-api/missing-image-deliveries";
+    private static final String POST_MISSING_IMAGE_CHD_ORDER_INCORRECT_URI = "/chd-order-api/missing-image-delivery";
+    private static final LocalDateTime DEFAULT_DATE = LocalDateTime.of(2019, 1, 1, 0, 0, 0);
+
+    @BeforeEach
+    void init() {
+        MissingImageDeliveryRequestApi missingImageDeliveryRequestApi = createMissingImageDeliveryRequestApi();
```

Comment:
You could also do 
private static final MissingImageDeliveryRequestApi MISSING_IMAGE_DELIVERY_REQUEST_API;

	static {
		MISSING_IMAGE_DELIVERY_REQUEST_API = new MissingImageDeliveriesDTO();
		MISSING_IMAGE_DELIVERY_REQUEST_API.setCompanyName("Test");
		MISSING_IMAGE_DELIVERY_REQUEST_API.setCompanyNumber("123");
		MISSING_IMAGE_DELIVERY_REQUEST_API.setFilingHistoryCategory("Test");
		MISSING_IMAGE_DELIVERY_REQUEST_API.setFilingHistoryDate("25-10-2018");
		MISSING_IMAGE_DELIVERY_REQUEST_API.setFilingHistoryDescription("Test");
		MISSING_IMAGE_DELIVERY_REQUEST_API.setId("Test");
		MISSING_IMAGE_DELIVERY_REQUEST_API.setItemCost("Test");
		MISSING_IMAGE_DELIVERY_REQUEST_API.setOrderedAt(LocalDateTime.now());
		MISSING_IMAGE_DELIVERY_REQUEST_API.setPaymentReference("Test");
	}

So you don't have to keep calling createMissingImageDeliveryRequestApi(). We've done it in quite a few of the apis.

Reply:
Nice, will implement now 👍